route/matcher_test.go

```diff
@@ -55,35 +60,6 @@ func TestGlobMatcher(t *testing.T) {
 	}
 }
 
-func TestGobwasGlobMatcher(t *testing.T) {
-	tests := []struct {
-		uri     string
-		matches bool
-		route   *Route
-	}{
-		// happy flows
-		{uri: "/foo", matches: true, route: &Route{Path: "/foo"}},
-		{uri: "/fool", matches: true, route: &Route{Path: "/foo?"}},
-		{uri: "/fool", matches: true, route: &Route{Path: "/foo*"}},
-		{uri: "/fools", matches: true, route: &Route{Path: "/foo*"}},
-		{uri: "/fools", matches: true, route: &Route{Path: "/foo*"}},
-		{uri: "/foo/x/bar", matches: true, route: &Route{Path: "/foo/*/bar"}},
-		{uri: "/foo/x/y/z/w/bar", matches: true, route: &Route{Path: "/foo/**"}},
-		{uri: "/foo/x/y/z/w/bar", matches: true, route: &Route{Path: "/foo/**/bar"}},
-
-		// error flows
-		{uri: "/fo", matches: false, route: &Route{Path: "/foo"}},
-		{uri: "/fools", matches: false, route: &Route{Path: "/foo"}},
-		{uri: "/fo", matches: false, route: &Route{Path: "/foo*"}},
-		{uri: "/fools", matches: false, route: &Route{Path: "/foo.*"}},
-		{uri: "/foo/x/y/z/w/baz", matches: false, route: &Route{Path: "/foo/**/bar"}},
-	}
-
-	for _, tt := range tests {
-		t.Run(tt.uri, func(t *testing.T) {
-			if got, want := gobwasGlobMatcher(tt.uri, tt.route), tt.matches; got != want {
-				t.Fatalf("got %v want %v", got, want)
-			}
-		})
-	}
-}
\ No newline at end of file
+func getRoute(path string) *Route {
```

Comment:
I'd prefer if you compile the glob matcher in the test since this keeps the changeset smaller, e.g.

```go
for _, tt := range tests {
    t.Run(tt.uri, func(t *testing.T) {
        tt.route.Matcher = glob.MustCompile(tt.route.path)
        ...
    })
}
```